.vscode/settings.json

```diff
@@ -0,0 +1,33 @@
+// Place your settings in this file to overwrite default and user settings.
+{
+  // Code spellcheck to ignore project-specific words
+  "cSpell.words": [
+    "commitlint",
```

Comment:
Some of these keywords, e.g. `hygen` are not used in this project.

Remove the needless words and do a spell-check on the existing files on the project. Then add the missing words here.

Note these words are case insensitive.